ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    batch_prediction_job_name = "[your-batch-prediction-job-name]"
</pre><p> </p><p>You can just supply a valid placeholder name. This won't be replaced during testing.</p><p>I forget if these have to be globally unique. If so, can you append a timestamp?</p><p> </p><pre class="ql-syntax" spellcheck="false">BATCH_PREDICTION_JOB_NAME = "text_classification_batch-" + TIMESTAMP
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Display names don't have to be globally unique.</p>

Reply:
<p>Actually, the job_name is going to be a resource ID, which the developer won't know (unless they do a list).</p>